types/ion-rangeslider/index.d.ts

```diff
@@ -0,0 +1,80 @@
+// Type definitions for ion-rangeslider 11.1
```

Comment:
The highest published package version for ion-rangeslider is 2.2, not 11.1.

Reply:
I take it the path change from `ion.rangeSlider` to `ion-rangeslider` is to reflect the published package name in NPM?